test/WebJobs.Script.Tests/Workers/Rpc/WebHostRpcWorkerChannelManagerTests.cs

```diff
@@ -176,8 +176,12 @@ public async Task SpecializeAsync_ReadOnly_KeepsProcessAlive(string runtime, str
             IRpcWorkerChannel workerChannel = CreateTestChannel(languageWorkerName);
 
             await _rpcWorkerChannelManager.SpecializeAsync();
-            Assert.True(testMetricsLogger.EventsBegan.Contains(MetricEventNames.SpecializationScheduleShutdownStandbyChannels)
-                && testMetricsLogger.EventsEnded.Contains(MetricEventNames.SpecializationScheduleShutdownStandbyChannels));
+            // Wait for debouce task to start
+            await TestHelpers.Await(() =>
+            {
+                return testMetricsLogger.EventsBegan.Contains(MetricEventNames.SpecializationScheduleShutdownStandbyChannels)
+                && testMetricsLogger.EventsEnded.Contains(MetricEventNames.SpecializationScheduleShutdownStandbyChannels);
+            }, pollingInterval: 6000);
```

Comment:
you can make this polling much faster -- 500? if there's a miss, there's no need to wait 6 seconds to check again.

Reply:
This is a debouce action that starts only after 5secs:  https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Workers/Rpc/WebHostRpcWorkerChannelManager.cs#L45

Trying to avoid polling sooner

Reply:
But polling this logger is really cheap, isn't it? I was thinking about the case that it doesn't come until 6.1 seconds, then we have to wait until 12 seconds. Just trying to make these fast -- we've got some tests taking over a minute right now and it creeps up.

Reply:
done!